composer.json

```diff
@@ -4,6 +4,9 @@
         "ext-gd": "0",
         "ext-curl": "0"
     },
+    "suggest": {
+        "ext-imagick": "to use the Imagick implementation",
```

Comment:
Read the article please. If it’s in your packages code, it’s not optional. You need to specify dependencies required to run 100% of the code, not the „happy path” or just the most common usage.

Instead, create another package, dependent on `ext-imagick` and `lbarulski/image-storage`, put your transformer there. Yeah, I know it’s a PITA.